server/webhook_worker.go

```diff
@@ -80,3 +83,44 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 
 	return nil
 }
+
+func (wh *webhook) FetchReporterNotification(bb []byte, reporter *jira.User) error {
+	if wh.eventTypes.ContainsAny("event_created_comment") || wh.eventTypes.ContainsAny("event_updated_comment") {
+		jwhook := &JiraWebhook{}
+		err := json.Unmarshal(bb, &jwhook)
+		if err != nil {
+			return err
+		}
```

Comment:
I think we can just use `wh.JiraWebhook` here, instead of creating a new `JiraWebhook`